test/Microsoft.Extensions.Logging.Test/LoggerFilterTest.cs

```diff
@@ -603,31 +603,6 @@ public void AddFilterForNonMatchingProviderDoesNotFilter()
         }
 
         [Fact]
```

Comment:
A bunch of code changed in this PR - should there not be more tests for this? E.g. the new extension methods, the new `UseConfiguration()` method, etc.